warp10/src/main/java/io/warp10/continuum/store/Store.java

```diff
@@ -150,61 +188,31 @@ public class Store extends Thread {
   private final long maxTimeBetweenCommits;
 
   /**
-   * Maximum size we allow the pending Puts list to grow
-   */
-  private final long maxPendingPutsSize;
-
-  /**
-   * Pool used to retrieve HTableInterface instances
-   */
-  //private final HTablePool htpool;
-
-  /**
-   * Connection to HBase
+   * Maximum size we allow the pending mutations list to grow
    */
-  private Connection conn;
-
-  /**
-   * Boolean indicating that we should recreate the connection to HBase
-   * due to IOErrors (such as regions which cannot be found due to region location cache not being updated)
-   */
-  private AtomicBoolean connReset = new AtomicBoolean(false);
-
-  /**
-   * HBase table where readings should be stored
-   */
-  private final TableName hbaseTable;
+  private final long maxPendingMutationsSize;
 
   private final boolean SKIP_WRITE;
 
   private int generation = 0;
 
   private UUID uuid = UUID.randomUUID();
 
-  private static Map<String,Connection> connections = new HashMap<String,Connection>();
-
-  /**
-   * Optional executor service for deletions
-   */
-  private ExecutorService deleteExecutor = null;
-
-  private final int nthreadsDelete;
+  //
+  // FoundationDB
+  //
 
-  private AtomicInteger inflightDeletions = new AtomicInteger(0);
-  private AtomicInteger deletionErrors = new AtomicInteger(0);
+  private final FDBContext fdbContext;
 
-  /**
-   * Rate limit to slow consumption down
-   */
-  private Double rateLimit = null;
+  private final long fdbRetryLimit;
 
-  private long throttlingDelay = 10000000L;
+  private final boolean FDBUseTenantPrefix;
 
   public Store(KeyStore keystore, final Properties properties, Integer nthr) throws IOException {
     this.keystore = keystore;
     this.properties = properties;
 
-    if ("true".equals(properties.getProperty("store.skip.write"))) {
+    if ("true".equals(properties.getProperty(Configuration.STORE_SKIP_WRITE))) {
```

Comment:
Could be replaced by
`this.SKIP_WRITE = "true".equals(properties.getProperty(Configuration.STORE_SKIP_WRITE));`